app/create_app.py

```diff
@@ -38,11 +42,26 @@ def inject_global_constants():
         return dict(
             stage="beta",
             service_title="Funding Service Design - Notification Hub",
-            service_meta_description=("Funding Service Design Iteration - Notification Hub"),
+            service_meta_description="Funding Service Design Iteration - Notification Hub",
             service_meta_keywords="Funding Service Design - Notification Hub",
             service_meta_author="DLUHC",
         )
 
+    executor = ContextAwareExecutor(
+        max_workers=Config.TASK_EXECUTOR_MAX_THREAD, thread_name_prefix="NotifTask", flask_app=flask_app
+    )
+    # Configure Task Executor service
+    task_executor_service = TaskExecutorService(flask_app=flask_app, executor=executor)
+    # Configurations for Flask-Apscheduler
+    scheduler = BackgroundScheduler()
+    scheduler.add_job(
+        func=scheduler_executor,
+        trigger="interval",
+        seconds=flask_app.config["SQS_RECEIVE_MESSAGE_CYCLE_TIME"],  # Run the job every 'x' seconds
```

Comment:
We usually access the config via the `Config` class so can you change this pls to `Config.SQS_RECEIVE_MESSAGE_CYCLE_TIME` pls

Reply:
Fixed thanks for noticing this